src/main/java/org/mifos/connector/channel/camel/routes/GSMAChannelRouteBuilder.java

```diff
@@ -178,8 +178,8 @@ public void configure() {
                     extraVariables.put(NOTE,gsmaChannelRequest.getDescriptionText());
                     logger.info("Payee Tenant ID {}", extraVariables.get("payeeTenantId"));
                     String bpmn = getWorkflowForTenant(tenantId);
-                    String tenantSpecificBpmn = bpmn.equals("default")?baseTransaction:bpmn
-                            .replace("{dfspid}", tenantId);
+                    String tenantSpecificBpmn = bpmn.equals("default")?baseTransaction .replace("{dfspid}",
+                            tenantId):bpmn.replace("{dfspid}", tenantId);
```

Comment:
we had this argument before that the tenant is not same as dfspid and dfspid will come as a separate header: accountholdingInstitutionId

Reply:
This dfspid is a placeholder in the bpmn name that we are replacing. If i change dfspid to accountholdinginstitutionid then i have to reformat all bpmn process id

Reply:
Upload bpmn logic works on this placeholder of dfspid so changing it here won't cause a major problem other than being in different from actual bpmn names

Reply:
let's track this major change separately. please create a ticket in the backlog for the same

Reply:
okay